evap/staff/views.py

```diff
@@ -285,6 +289,28 @@ def semester_import(request, semester_id):
 
 
 @staff_required
+def semester_export(request, semester_id):
+    semester = get_object_or_404(Semester, id=semester_id)
+
+    ExportSheetFormset = formset_factory(form=ExportSheetForm, can_delete=True, extra=1)
+    formset = ExportSheetFormset(request.POST or None, form_kwargs={'semester': semester})
+
+    if formset.is_valid():
+        ignore_not_enough_answers = request.POST.get('ignore_not_enough_answers', False)
+        course_types = []
+        for form in formset:
+            course_types.append(form.cleaned_data.get('selected_course_types'))
+
+        filename = "Evaluation-%s-%s.xls" % (semester.name, get_language())
```

Comment:
are you aware that the semester name is localized as well? so it would be "sommersemester 13-de" and "summer term 13-en". also, the spaces look a bit weird to me, or rather, that there are none around the dashes.